### ISSUE TITLE
C++模板元-质数序列 | 594飞飘的个人博客

### ISSUE THREAD

**Issue by feipiao594** [feipiao594/feipiao594.github.io#53]
https://feipiao594.github.io/posts/3187765d.html#more 

 C++模板元-质数序列花了一点时间巩固自己的模板元编程，写了一个编译期质数数组生成器

**Comment by ArcticLampyrid** [comment]
use `-ftemplate-depth` to relax the limitation for gcc

**Comment by feipiao594** [comment]
> @ArcticLampyrid
> use `-ftemplate-depth` to relax the limitation for gcc

yes,I know.I mean reducing the number of iterations to optimize.After all, a high number of iterations is not good
